extensions/wikia/AdEngine/js/slot/scrollHandler.js

```diff
@@ -9,21 +9,32 @@ define('ext.wikia.adEngine.slot.scrollHandler', [
     'use strict';
 
     var logGroup = 'ext.wikia.adEngine.slot.scrollHandler',
-        isRefreshed = {
-            PREFOOTER_LEFT_BOXAD: false,
-            PREFOOTER_RIGHT_BOXAD: false
-        },
-        reloadedView = {
-            PREFOOTER_LEFT_BOXAD: 0,
-            PREFOOTER_RIGHT_BOXAD: 0
-        };
+        isRefreshed = {},
+        reloadedView = {},
+        context = adContext.getContext(),
+        config = context.opts.scrollHandlerConfig ||
+            {
+                PREFOOTER_LEFT_BOXAD: { reloadedViewMax: 1 },
+                PREFOOTER_RIGHT_BOXAD: { reloadedViewMax: 3 }
+            };
 
     function init() {
-        if (adContext.getContext().opts.enableScrollHandler) {
+        if (context.opts.enableScrollHandler) {
+            for (var slotName in config) {
+                if (config.hasOwnProperty(slotName)) {
+                    isRefreshed[slotName] = false;
```

Comment:
@fraszczakszymon I would like to keep in config only the things which are really configurable. `isRefreshed` and `reloadedView ` will be the same for all slots from the start point.
If you think it will help to increase readability or something, I can move it there :smile: